.eslintrc

```diff
@@ -63,6 +64,7 @@
     "react/jsx-indent-props": ["error", 2],
     "react/jsx-max-props-per-line": ["error", { "maximum": 1 }],
     "react/jsx-equals-spacing": ["error", "never"],
+    "react/jsx-first-prop-new-line": ["off"],
```

Comment:
upgrading babel and eslint which seems to be required to work with node 6
also meant upgrading the airbnb preset which seems to have changed a bunch
of stuff. I'm trying to figure out what is needed and what isn't right now.

On Thu, May 19, 2016 at 5:01 PM, Victor Porof <notifications@github.com>
wrote:

> In .eslintrc
> <https://github.com/mozilla/tofino/pull/478#discussion_r63972724>:
>
> > @@ -63,6 +64,7 @@
> >      "react/jsx-indent-props": ["error", 2],
> >      "react/jsx-max-props-per-line": ["error", { "maximum": 1 }],
> >      "react/jsx-equals-spacing": ["error", "never"],
> > +    "react/jsx-first-prop-new-line": ["off"],
>
> Interesting that we need this
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/tofino/pull/478/files/64cde3f78d0fe207f1f2b35ca0a2a0ae82b44ce1#r63972724>
>